p2p/test/transport/deadline_test.go

```diff
@@ -0,0 +1,131 @@
+package transport_integration
+
+import (
+	"context"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/libp2p/go-libp2p/core/network"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewStreamDeadlines(t *testing.T) {
+	for _, tc := range transportsToTest {
+		t.Run(tc.Name, func(t *testing.T) {
+			if strings.Contains(tc.Name, "WebSocket") || strings.Contains(tc.Name, "Yamux") {
+				t.Skip("Yamux does not adhere to deadlines: https://github.com/libp2p/go-yamux/issues/104")
```

Comment:
This issue was resolved in https://github.com/libp2p/go-yamux/pull/105. Should this PR be updated?

Reply:
We have to release that fix https://github.com/libp2p/go-yamux/pull/106